recipes/r-longitudinaldata/meta.yaml

```diff
@@ -0,0 +1,81 @@
+{% set version = '2.4.1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-longitudinaldata
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/longitudinalData_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/longitudinalData/longitudinalData_{{ version }}.tar.gz
+  sha256: 6d8b155726b5e1913711c84af3e0fc56bb3a1097e0471628572b88239f63db52
+
+build:
+  skip: True  # [osx]
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{posix}}zip               # [win]
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - {{ cdt('mesa-dri-drivers') }}  # [linux]
+    - {{ cdt('libselinux') }}  # [linux]
+    - {{ cdt('libxdamage') }}  # [linux]
+    - {{ cdt('libxxf86vm') }}  # [linux]
+  host:
+    - r-base
+    - r-class
+    - r-clv
+    - r-misc3d
+    - r-rgl
+    - xorg-libxfixes  # [linux]
+  run:
+    - r-base
+    - r-class
+    - r-clv
+    - r-misc3d
+    - r-rgl
+    - xorg-libxfixes  # [linux]
```

Comment:
Would this succeed if you move it to `test: requires: xorg-libxfixes  # [linux]`? We can try that also after feedstock conversion.

Reply:
Good question, we should try that.